CHANGELOG.md

```diff
@@ -12,6 +12,17 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 - Error handling is now performed with the `anyhow` crate instead of `failure`.
   (#1805)
 
+- APIs which previously used futures from the `futures 0.1` crate are now
+  made `async`, or are using `Future`s from the standard library. (#1796, #1804)
+  The main affected APIs are as follows:
+  
+  - Endpoint definitions in the `exonum-api` crate and their counterparts
+    in `exonum-rust-runtime`
+  - Transaction sending with `ApiSender` from the `exonum` crate
+  - Node start-up with `Node::run` (`exonum-node` crate) and `NodeBuilder::run`
+    (`exonum-cli` crate), and shutdown with `ShutdownHandle` (`exonum-node` crate)
+  - HTTP API testing with `TestKitApi` (`exonum-testkit` crate)
```

Comment:
I think that we should also add remark about `Broadcaster` and their blocking twin.